types/react/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for React 18.0
+// Type definitions for React 18.3
```

Comment:
Is `inert` only available in React 18.3?

Reply:
Please understand that this PR is in draft and waiting for a resolution of https://github.com/facebook/react/pull/24730